addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/PolicyDefinition.java

```diff
@@ -44,7 +43,7 @@ public class PolicyDefinition extends AutomationData {
     private String defaultThreshold = JobUtils.thresholdToI18n(AlertThreshold.MEDIUM.name());
     private List<Rule> rules = new ArrayList<>();
 
-    public static void parsePolicyDefinition(
+    public void parsePolicyDefinition(
```

Comment:
Worth doing in its own commit as we should also remove the `policyDefinition` parameter.